### ISSUE TITLE
Add option simulate_psf to keep the output dir

### ISSUE THREAD

**Issue by adonath** [cxcsds/ciao-contrib#740]
Currently `simulate_psf` seem to always cleanup the output dir. However it would be useful to keep the intermediate files to run e.g. `marx2fits` independently again with different parameters, without re-simulating the events. Maybe this is possibly already now, however I could no figure out how.

The code seems to always call: https://github.com/cxcsds/ciao-contrib/blob/main/bin/simulate_psf#L861

What I would like to achieve is that I can use `simulate_psf` to run the simulation and the independently call `marx2fits` after. However this only seems to be possible by also running `marx` independently (not via `simulate_psf`).

**Comment by adonath** [comment]
Or phrasing the question differently: is there an option to run `simulate_psf simulator=file` with files produced by `marx`?

**Comment by kglotfelty** [comment]
Try using

```bash
pset srcflux keepiter=yes
```

**Comment by kglotfelty** [comment]
In answer to  the question "simulate_psf simulator=file with files produced by marx?" the answer is no.

The marx output eg before running marx2fits is already complete.  All that is left is to run marx2fits ; and all it is doing is converting from marx's internal format to FITS format.

**Comment by adonath** [comment]
Thanks @kglotfelty! This I have tried, but it will only keep the projected files, parameter files and images:

```
-rw-r--r--  1 axel  staff   132K Mar  8 15:32 marx.psf
-rw-r--r--  1 axel  staff   129K Mar  8 15:32 marx_i0000.psf
-rw-r--r--  1 axel  staff    26K Mar  8 15:32 marx_i0000_marx.par
-rw-r--r--  1 axel  staff    82K Mar  8 15:32 marx_i0000_projrays.fits
-rw-r--r--  1 axel  staff   129K Mar  8 15:32 marx_i0001.psf
-rw-r--r--  1 axel  staff    26K Mar  8 15:32 marx_i0001_marx.par
-rw-r--r--  1 axel  staff    79K Mar  8 15:32 marx_i0001_projrays.fits
-rw-r--r--  1 axel  staff    98K Mar  8 15:32 marx_projrays.fits

```

> In answer to the question "simulate_psf simulator=file with files produced by marx?" the answer is no.

Thanks! But does this mean I always need to rerun the full event simulation when e.g. just changing the `blur` parameter? Is there any way to avoid this?

**Comment by adonath** [comment]
Here is another use case:

When using `simulate_psf pileup=yes` the alpha parameter of the pileup model seems hardcoded. If I'd like to change it I would need to run `marxpileup` independently, for this I need again the data from the output directory.

**Comment by kglotfelty** [comment]
Yes, to change the blur you have to re-run marx, it corresponds the `AspectBlur` parameter in `marx.par`.

With `keepiter=yes` you get the marx parameter file(s) so you can re-create the same marx directories by just doing

```bash
marx @@marx_i0001_marx.par
```
so you can re-run marxpileup with your choice of parameter settings.

**Comment by adonath** [comment]
Thanks @kglotfelty, that sounds like and acceptable compromise.

**Comment by DougBurke** [comment]
Do we want to add an example describing something like this to the help file?

**Comment by kglotfelty** [comment]
An example was added to the ahelp as part of #741

**Comment by adonath** [comment]
Thanks @kglotfelty and @DougBurke!
